ob-pattern-1/values.yaml

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+wso2:
+  # MySQL datasource configurations
+  #   enabled:          The configuration should be set to be 'true' if a MySQL database should be spawned as a pod within the cluster
+  #   host:             Hostname of the MySQL database
+  #   username:         MySQL database username
+  #   password:         MySQL database password
+  #   driverClass:      JDBC Driver class name
+  #   validationQuery:  SQL validation query
+  db:
```

Comment:
@kdanW shall we configure this setup to work with MySQL within this PR itself before finalizing this it. Having these parameters and having no use of these does not look right, IMO.

Reply:
Sure. Might take sometime,but as I have finished Docker testing earlier than I thought, I can start working on this

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/7d02d48ffa7b4893cf58076f2808e61124c02869